src/reducers/Task.js

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const Task = createSlice({
+  name: 'task',
+  initialState: {
+    items: [
+      {
+        id: '19asdfiuhas0823hjkadsh809',
+        content: 'New to-do',
+        isChecked: false
+      },
+      {
+        id: '19asdfiuhas0823hjkadsh809',
+        content: 'Water my Plants',
+        isChecked: false
+      },
+      {
+        id: '19asdfiuhas0823hjkadsh809',
+        content: 'Trim monstera',
+        isChecked: false
```

Comment:
We should definitely keep the todo functionality, it's a great feature of the app